src/encoded/static/components/regulome_search.js

```diff
@@ -341,7 +343,7 @@ const NearbySNPsDrawing = (props) => {
     if (context.nearby_snps && context.nearby_snps[0] && context.nearby_snps[0].coordinates) {
         startNearbySnps = +context.nearby_snps[0].coordinates.lt;
         endNearbySnps = +context.nearby_snps[context.nearby_snps.length - 1].coordinates.lt;
-        coordinate = +Object.keys(context.variants)[0].split('-')[1];
+        coordinate = +context.query_coordinates[0].split('-')[1];
```

Comment:
I understand you using this to maintain consistency, but it is quite confusing. It is normally written as: 
coordinate =+ context.query_coordinates[0].split('-')[1];

Reply:
Should I fix others up there to really maintain consistency?